app/views/searches/show.html.erb

```diff
@@ -9,7 +9,7 @@
   <div class="govuk-form-group">
     <%= f.button name: nil, class: "request-button" do %>
       Search
-      <%= "again" if @search %>
+      <%= "again" if @search.search&.present? %>
```

Comment:
`@search.search` reads a bit odd to me, I don't feel strongly about it but `@search.term` or `@search.text` is more readable IMO

Reply:
Yeh, I think I'll keep it as is in this PR but I'll probably tweak this in the future - I'd like searches to be something like `/search?q=my+search+term`